### PR TITLE
[No QA] Remove unused methods and exports

### PR DESCRIPTION
**Pull request by tgolen**
This is just some cleanup that I came across

### Tests / QA Steps
No QA - Normal regression steps will catch anything

### Tested On

- [x] Web
- [x] Mobile Web
- [x] Desktop
- [x] iOS
- [x] Android

#### Web
image

#### Mobile Web
image

#### Desktop
image

#### iOS
image

#### Android
image

### CODE REVIEW

File: src/libs/Navigation/AppNavigator/AuthScreens.js [hunk 1]
```diff
@@ -119,7 +119,6 @@ class AuthScreens extends React.Component {
         User.getBetas();
         fetchAllReports(true, true);
         fetchCountryCodeByRequestIP();
-        BankAccounts.fetchBankAccountList();
```
Comment (Jag96) [inline src/libs/Navigation/AppNavigator/AuthScreens.js:122]: Looks like the `BankAccounts` import at the top of the file can also be removed

**Review by Jag96 — CHANGES_REQUESTED** [review]
Lint is failing b/c of an unused import

**Review by pecanoro — DISMISSED** [review]

**Comment by tgolen** [comment]
Ooops, updated to remove the import.

**Review by Jag96 — APPROVED** [review]
LGTM

**Pull request merged**